### PR TITLE
Multiple mentioning issue fixed in chat

### PR DESCRIPTION
**Pull request by SachinL98**
When multiple times mentioned a same user in the chat, all occurrences must be displayed as name.

### CODE REVIEW

**Review by gihansdp — CHANGES_REQUESTED** [review]
Since the linked ticket is from saas project, update the saas project to install the `vue-advanced-chat` from this branch using and another PR. Then we can test this change on saas project.

**Comment by gihansdp** [comment]
You need to build this for changes to take effect as a library. Run `npm run build` on root folder.

**Comment by SachinL98** [comment]
Done

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
 	"dependencies": {
 		"linkifyjs": "^2.1.9",
 		"v-click-outside": "^3.1.2",
+		"vue-advanced-chat": "^2.1.0",
```
Comment (gihansdp) [inline package.json:74]: What's this? `vue-advanced-chat` require `vue-advanced-chat`?

File: demo/src/firestore/index.js [hunk 2]
```diff
@@ -3,10 +3,16 @@ import 'firebase/firestore'
 import 'firebase/database'
 import 'firebase/storage'
 
-const config = 
-	process.env.NODE_ENV === 'development'
-		? JSON.parse(process.env.VUE_APP_FIREBASE_CONFIG)
-		: JSON.parse(process.env.VUE_APP_FIREBASE_CONFIG_PUBLIC)
+const config = {
```
Comment (gihansdp) [inline demo/src/firestore/index.js:6]: Do not commit your keys. Keep it as it is to get from the env

**Review by gihansdp — APPROVED** [review]

**Pull request merged**